bedevere/backport.py

```diff
@@ -15,23 +15,46 @@
                     '[{branch} branch](https://github.com/python/cpython/tree/{branch}).')
 
 
+async def _copy_over_labels(gh, original_issue, backport_issue):
+    """Copy over relevent labels from the origial PR to the backport PR."""
+    label_prefixes = "skip", "type", "sprint"
+    labels = list(filter(lambda x: x.startswith(label_prefixes),
+                    util.labels(original_issue)))
+    if labels:
+        await gh.post(backport_issue["labels_url"], data=labels)
+
+
+async def _remove_backport_label(gh, original_issue, branch, backport_pr_number):
+    """Remove the appropriate "backport to" label on the original PR.
+
+    Also leave a comment on the original PR referencing the backport PR.
+    """
+    backport_label = BACKPORT_LABEL.format(branch=branch)
+    if backport_label not in util.labels(original_issue):
+        return
+    await gh.delete(original_issue['labels_url'], {'name': backport_label})
+    message = MESSAGE_TEMPLATE.format(branch=branch, pr=backport_pr_number)
+    await gh.post(original_issue['comments_url'], data={'body': message})
+
+
 @router.register("pull_request", action="opened")
 @router.register("pull_request", action="edited")
-async def remove_backport_label(event, gh, *args, **kwargs):
-    title = util.normalize_title(event.data['pull_request']['title'],
-                                 event.data['pull_request']['body'])
+async def manage_labels(event, gh, *args, **kwargs):
+    if event.data["action"] == "edited" and "title" not in event.data["changes"]:
+        return
+    pull_request = event.data["pull_request"]
+    title = util.normalize_title(pull_request['title'],
+                                 pull_request['body'])
     title_match = TITLE_RE.match(title)
     if title_match is None:
         return
     branch = title_match.group('branch')
-    backport_label = BACKPORT_LABEL.format(branch=branch)
-    pr_to_backport = title_match.group('pr')
-    backported_pr = event.data['number']
+    original_pr_number = title_match.group('pr')
 
-    issue_to_backport = await gh.getitem(event.data['repository']['issues_url'],
-                                         {'number': pr_to_backport})
-    if backport_label not in util.labels(issue_to_backport):
-        return
-    await gh.delete(issue_to_backport['labels_url'], {'name': backport_label})
-    message = MESSAGE_TEMPLATE.format(branch=branch, pr=backported_pr)
-    await gh.post(issue_to_backport['comments_url'], data={'body': message})
+    original_issue = await gh.getitem(event.data['repository']['issues_url'],
+                                         {'number': original_pr_number})
```

Comment:
Should the `{` line up with the `(` from the previous line?

Reply:
Yep 😄